source/_integrations/matter.markdown

```diff
@@ -104,10 +104,17 @@ Each Matter network is called a fabric. Each home automation controller that con
 
 ### Prerequisites
 
+- Make sure you have the latest version of Home Assistant installed.
 - On the device packaging, check for both the Matter logo and for either the Wi-Fi or the Thread logo.
 - Check if the QR code is only on the packaging or if it is also on the device.
-  - If it is only on the packaging, snap a picture of the QR code and the device and store it in a save place.
+  - If it is only on the packaging, snap a picture of the QR code and the device and store the image and the numerical code in a save place.
   - If you lose the QR code and disconnect the device at some point, you won't be able to connect to that device again without the QR code.
+- Check your network settings:
+  - Make sure IPv6 is enabled in your home network. You do not need IPv6 from your <abbr title="internet service provider">ISP</abbr>, just in your local network.
+  - Make sure IPv6 is enabled in Home Assistant:
+    - Go to {% my network title="**Settings** > **System** > **Network**" %}, and under **Configure network interfaces**, make sure that IPv6 is enabled.
+  - Avoid using Matter in a complex network setup with vLANS or mDNS responders. These are more likely to cause network issues than if you use Matter in a flat network.
```

Comment:
this one is also in the troubleshooting section - can we do something smart with just linking there. Something like "Make sure your home network meets the requirements" --> link to the troubleshooting or advanced details

Reply:
haha, right. that's why I didn't include it the first time round. Already forgot. 

Linking: I don't like the idea of linking to the troubleshooting section here, though:
- If we think the majority of people will need to look into their network settings before starting, this is a true prereq and should be mentioned here. 
- If not, then it is better not to add a link that sends them off-track.

Reply:
removed.